migrations/versions/9f5a6dbfec37_add_vbr_fields.py

```diff
@@ -0,0 +1,33 @@
+"""
+1.32.0 add vbr fields
```

Comment:
```suggestion
1.26.0 add vbr fields
```
The last one was at 1.25.0